### PR TITLE
[RedirectBundle] Configure redirect admin list configurator as service

### PR DESCRIPTION
**Pull request by dannyvw**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | comma separated list of tickets fixed by the PR

We needed some custom logic in this class. Do I need to add some BC layer?

### CODE REVIEW

**Review by acrobat — CHANGES_REQUESTED** [review]
This will indeed need a BC layer. I think something like in the translation controller can be implemented in a BC way

https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/cc640556e5cac9874ea021a0a897c00114ff590c/src/Kunstmaan/TranslatorBundle/Controller/TranslatorController.php#L246-L260

File: src/Kunstmaan/RedirectBundle/Resources/config/services.yml [hunk 1]
```diff
@@ -22,8 +22,14 @@ services:
...
         calls:
             - setContainer: [ ] # Container service locator will be injected filled by the services from subscribed services
         tags: ['controller.service_arguments', 'container.service_subscriber']
 
+    Kunstmaan\RedirectBundle\AdminList\RedirectAdminListConfigurator:
+        arguments:
+            $em: '@doctrine.orm.entity_manager'
+            $domainConfiguration: '@kunstmaan_admin.domain_configuration'
+        autowire: true
+        autoconfigure: true
```
Comment (acrobat) [inline src/Kunstmaan/RedirectBundle/Resources/config/services.yml]:
```suggestion
            - '@doctrine.orm.entity_manager'
            - '@kunstmaan_admin.domain_configuration'
```

Won't use autowiring/autoconfigure in bundles as the service config should be explicit, also don't use named arguments as the variables names are not included in the bc policy

**Review by acrobat — CHANGES_REQUESTED** [review]

File: src/Kunstmaan/RedirectBundle/Resources/config/services.yml (hunk 1 above)
Comment (dannyvw) [reply src/Kunstmaan/RedirectBundle/Resources/config/services.yml]: @acrobat Done

**Review by acrobat — APPROVED** [review]

**Comment by acrobat** [comment]
Thanks @dannyvw!

**Pull request merged**